pywps/dblog.py

```diff
@@ -170,13 +173,16 @@ def get_session():
                 engine = sqlalchemy.create_engine(database,
                                                   echo=echo,
                                                   connect_args={'check_same_thread': False},
-                                                  poolclass=NullPool)
```

Comment:
Do you really need a `QueuePool` here? I believe you definitely need to keep `NullPool`, because the multiprocessing forks, and can lead to open/broken connections. I believe there will basically be no performance impact for this.

Reply:
See: https://github.com/geopython/pywps/commit/203ae6dc83f46d783852a3b72b693a5d2301f453

Reply:
@davidcaron  `NullPool` will have a penalties performance if you use PostGRESQL, here should be `NullPool` to avoid thread problems

Reply:
I believe, this is just for the sqlite, as @jorgejesus is writing, we are constantly hitting threads problem with sqlite (any best-practice solution would be great)